contracts/pcv/balancer/BalancerPool2Lens.sol

```diff
@@ -119,6 +119,7 @@ contract BalancerPool2Lens is IPCVDepositBalances {
         }
         if (feiInPair) {
             uint256 otherReserves = _getIdealReserves(balances, prices, weights, j);
+            otherReserves = (otherReserves * bptsOwned) / totalSupply;
```

Comment:
Worthwhile adding a unit or integration test to validate this fix? The fix is simple enough